### ISSUE TITLE
Add appropriate aria elements

### ISSUE THREAD

**Issue by kentcdodds** [brophdawg11/remix-validity-state#3]
This should have `aria-invalid` and `aria-describedby` which points to the errors element.

For example: https://github.com/remix-run/indie-stack/blob/2d7b4768ad419896bbb899d75ac4cb71c1f6aaa0/app/routes/login.tsx#L102-L110

**Comment by brophdawg11** [comment]
Is that recommended over the [`aria-errormessage`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-errormessage) being used currently?

**Comment by kentcdodds** [comment]
I didn't notice that prop somehow. I think that is probably even better
